scrapers/TeasePOV

```diff
@@ -0,0 +1,25 @@
+name: TeasePOV
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - teasepov.com/videos/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="slideTitle"]
+      Date:
+        selector: //div[@class="slideDate"]
+        postProcess:
+          - replace:
+              - regex: (?:[^A-Z]+)(\w+)(?:.)(\d+)(?:\D+)(\d+)
+                with: $3-$1-$2
+          - parseDate: 2006-January-02
+          - parseDate: 2006-Jan-02
+          - parseDate: 2006-January-2
+      Details:
+        selector: //div[@class="videoDescription"]
+      Studio:
+        Name:
+          fixed: TeasePOV
+# Last Updated Februaru 19, 2022
```

Comment:
Just a suggestion to clean up the date and add a scene photo.
```name: TeasePOV
sceneByURL:
  - action: scrapeXPath
    url:
      - teasepov.com
    scraper: sceneScraper
xPathScrapers:
  sceneScraper:
    scene:
      Title: //div[@class="slideTitle"]
      Date:
        selector: //div[@class="slideDate"]
        postProcess:
          - replace:
              - regex: "/"
                with: ""
          - parseDate: January 2, 2006
      Details:
        selector: //div[@class="videoDescription"]
      Studio:
        Name:
          fixed: TeasePOV
      Image:
        selector: //div[@id="player"]/img/@src    
# Last Updated February 20, 2022

Reply:
I'm sorry, I'm new to using github. I think your changes are great. Am I supposed to do something to make your changes the final piece? Because I'm not sure how to.
Thanks

Reply:
Nice :-) , Well done yeah once you get the hang of github you will be fine, I am still learning myself everyday